relay-server/src/utils/metrics_rate_limits.rs

```diff
@@ -51,13 +51,13 @@ impl BucketLimiter {
                 }
 
                 if mri.name == "duration" {
-                    // The "duration" metric is extracted exactly once for every processed transaction,
-                    // so we can use it to count the number of transactions.
+                    // The "duration" metric is extracted exactly once for every processed
+                    // transaction, so we can use it to count the number of transactions.
```

Comment:
Since this PR also implements the renaming dance of transactions, indexed transactions, and processed transactions:
```suggestion
                    // The "duration" metric is extracted exactly once for every
                    // transaction, so we can use it to count the number of transactions.
```